app/square_size.go

```diff
@@ -5,9 +5,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// GovSquareSizeUpperBound returns the maximum square size that can be used for a block
-// using the governance parameter blob.GovMaxSquareSize.
-func (app *App) GovSquareSizeUpperBound(ctx sdk.Context) int {
+// GetMaxSquareSize returns the maximum square size that can be used for a
+// block.
+func (app *App) GetMaxSquareSize(ctx sdk.Context) int {
```

Comment:
[discussion]
I forget what was the agreed upon scheme for naming. Per this PR, the term Max square size is now referring to current state of the largest square for a given height, which I believe differs from how this  was used in past (or at least how I've used this term in the past). 

That's not to say that this is a bad change! just a departure

Reply:
ref https://github.com/celestiaorg/celestia-app/blob/main/docs/architecture/adr-021-restricted-block-size.md#decision

Reply:
Instead of trying to come up with a unique third name, I briefly explored what it would be like if `MaxSquareSize` actually acted as an upper bound for `GovMaxSquareSize` so that we could use `GovMaxSquareSize` throughout the app as the real max square size. 

See https://github.com/celestiaorg/celestia-app/issues/2185

It turns out we can't do that because the function signature of parameter validation functions 😑